platform/util/platform-util-unavailableurls/src/test/java/org/codice/ddf/platform/util/http/UnavailableUrlsTest.java

```diff
@@ -27,11 +32,17 @@
 
 public class UnavailableUrlsTest {
 
+    private static final String MAX_TIMEOUT_SECONDS = "300";
+
+    private static final String INITIAL_TIMEOUT_SECONDS = "10";
+
     private ScheduledExecutorService scheduler;
 
     @Before
     public void setUp() throws Exception {
         scheduler = Mockito.mock(ScheduledExecutorService.class);
+        System.setProperty(MAX_TIMEOUT_SECONDS_PROPERTY, MAX_TIMEOUT_SECONDS);
+        System.setProperty(INITIAL_TIMEOUT_SECONDS_PROPERTY, INITIAL_TIMEOUT_SECONDS);
```

Comment:
Modifying the environment in unit tests is not hermetic. You could get the initial values (if any) and reset them in your `@After`, but that's still not hermetic for the duration the test runs.

I would suggest refactoring the unit under test slightly so that you can inject a mock system properties map. This keeps the test hermetic without compromising the integrity of the production code.